LayoutTests/editing/selection/user-select-none-02.html

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<title>Test user-select: none Test</title>
+<style>
+#test-element {
+   background-color: lightblue;
+   padding: 10px;
+   font-size: 16px;
+   text-align: center;
+   border: 1px solid #000;
+   -webkit-user-select: none;
+}
+</style>
+</head>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
`js-test.js` is defacto going forward (or `testharness.js` similar to WPT). So use it instead of `js-test-pre.js`.